package.json

```diff
@@ -10,7 +10,8 @@
     "dev": "webpack-cli serve --mode development ",
     "build": "webpack --mode production",
     "start": "npm run build & serve -s ./dist ",
-    "coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls"
+    "coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
+    "compile:sass": "node-sass sass/index.scss css/index.css -w"
```

Comment:
Add a rule in your weback file config for styles compilation. I don't know if you'd want to run this script on every development change.

Reply:
I was thinking of putting the compile:sass in the script of dev, but let me work with the webpack configuration!.

Reply:
done!.

Reply:
So do you still need this script?